Dockerfile

```diff
@@ -17,7 +17,10 @@ ENV SBT_URL "https://github.com/sbt/sbt/releases/download/v${SBT_VERSION}/sbt-${
 
 RUN set -o pipefail && \
     apk update && \
+    apk add --upgrade apk-tools && \
+    apk upgrade --available && \
     apk add --no-cache --update bash wget npm git openssh && \
+    npm install -g npm@8.5.1 && \
```

Comment:
nit? why not pin to @latest ?

Reply:
on the theory that explicit versioning is better than implicit? i could be convinced otherwise

Reply:
nah this is all good just curious